nautobot/core/management/commands/startplugin.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+from django.conf import settings
+from django.core.management.templates import TemplateCommand
+
+
+class Command(TemplateCommand):
+    help = (
+        "Creates a Nautobot Plugin directory structure for the given app name in "
+        "the current directory or optionally in the given directory."
+    )
+    missing_args_message = "You must provide an plugin name."
+
+    def add_arguments(self, parser):
+        default_template = os.path.join(settings.BASE_DIR, "core/templates/plugin_template")
```

Comment:
Though we don't support Windows installation at present, may as well be technically more correct:

```suggestion
        default_template = os.path.join(settings.BASE_DIR, "core", "templates", "plugin_template")
```

Reply:
Where is the `"core<span class...` coming from? Is it just github?

Reply:
That said, I see what you mean.

Reply:
Not sure if I'm answering your question or not but definitely will want to check that a wheel installation (`poetry build`, then install the `.whl` into a virtualenv) includes the template files correctly in its installation.

Reply:
No, it was the html that was showing up in github, but now it is not showing, so I think there was some funk going on.  That said, I am testing what you just posted now.